app/design/frontend/Magento/luma/Magento_SendFriend/web/css/source/_module.less

```diff
@@ -44,3 +44,24 @@
         }
     }
 }
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__m) { 
+    .form.send.friend {
+        .fieldset {
+            .field {
+                .control {
+                    width: 100%;
+                }  
+            }
+            padding-bottom: 10px;
```

Comment:
1) should be move to top just after class name because else it's hard to read/understand for what selector it's ment.
2) As I understand it's needed for remove botton positioning. Could we instead use this padding for selector ".additional" as it's not needed for each input field. Also change 10px to variable @indent__xs

Reply:
Ok, Thankyou !

Reply:
>> Could we instead use this padding for selector ".additional" as it's not needed for each input field.
Reply : Its getting issues If I am applying padding and removing position.